### PR TITLE
Add readme details for Mac OS

### LINKED ISSUE #14: Add Readme (Mac OS)

### PR DESCRIPTION
**Pull request by Rwothoromo**
#### What does this PR do?
Adds a `README` for easy setup.

#### Description of Task to be completed?
- [x] Add instructions for all users to get started quickly.
- [x] Add Contributors file.

#### How should this be manually tested?
Follow the instructions on the `README`.

#### Any background context you want to provide?
Vital README instructions were missing.

fixes #14

### CODE REVIEW

**Comment by tibetegya** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -1,2 +1,33 @@
...
+The official website for Python Kampala (PyKla)
+
+## Technologies
+
+* Python 3.6
+* SQLite3
+
+## Requirements
+
+* Install [Python](https://www.python.org/downloads/)
```
Comment (lubegamark) [inline README.md]: You could Add what you've put under a Windows section. Someone else will add a Unix sepcific section later.

**Review by lubegamark — CHANGES_REQUESTED** [review]
This is very nice. Just separate the content into windows Sections and we'll be good to go.

File: README.md (hunk 1 above)
Comment (Rwothoromo) [reply README.md]: Hello @lubegamark, thank you for the feedback. I think I will just add details for Mac OS.

**Pull request merged**